tests/spec/core/markdown-spec.js

```diff
@@ -417,4 +419,25 @@ describe("Core - Markdown", () => {
       expect(preText[2]).toBe("     party()");
     });
   });
+
+  describe("Markdown-inside-block backward compatibility", () => {
+    it("parses indented <pre> after a list", async () => {
+      const idl = `dictionary Indented {\n  any shouldBeIndented();\n};`;
+      const body = `
+        <section>
+        1. I lack double newlines between HTML block
+        </section>
+        <pre class="idl" id="pre">
```

Comment:
Just to confirm, triple backtick blocks work fine?

Reply:
Yup, as triple backtick requires adding an extra newline after any HTML tags.